building-a-preview/index.md

```diff
@@ -0,0 +1,777 @@
+3. Press the **OK** button.
+
+That preview will be moved to the **Base Preview** section of the Repository
+Dashboard. From now on, Previews will build from the snapshot created when the
+Base Preview was built.
+
+1. Go to the **Manage Base Previews** link on the Repository Dashboard.
+
+![Base Preview Selection](_images/base-preview-before.png)
+
+2. Click the checkbox next to the Preview you want to use as a Base Preview.
+3. Press the **OK** button.
+
+![Base Preview Selection](_images/base-preview-select.png)
+
+That preview will be moved to the **Base Preview** section of the Repository
+Dashboard.
+
+![Base Preview Selection](_images/base-preview-after.png)
+
+That's it! From now on, Previews will build from the snapshot created when the
+Base Preview was built.
+
+If you're ever wondering which Base Preview was used when generating a Preview,
+just look under the name of the Preview, at the "from _Base Preview Name_":
+
+![View Base Preview for Preview](_images/biew_base_preview_for_preview.png)
+
+### Keeping Base Previews updated
+
+You'll generally want to keep your Base Preview up to date with your latest
+codebase, and a fresh copy of your database, image files, and other assets. By
+default, Tugboat automatically checks for updates every night at 12 am ET, and
+[refreshes](#refresh) your Base Preview with these updates. To change this,
+check the Repository Settings.
+
+Tugboat performs this update by pulling the latest code from git for the branch
+or Tag the preview was built from. During the update, Tugboat runs any commands
+in the `update` and `build` sections for services in the
+[configuration](../setting-up-services/index.md#create-a-tugboat-config-file)
+file.
+
+### Using multiple Base Previews
+
+Tugboat allows multiple Base Previews to be defined. The effect of doing this is
+that every preview will generate the corresponding number of Base Preview
+derivatives. So, if you have three Base Previews defined, and submit a pull
+request for Tugboat to build a preview, you will end up with three previews for
+that pull request, each starting from a different Base Preview.
+
+This feature allows you to test code, for instance, against different PHP
+versions, database content, etc.
+
+![Multiple Base Previews generating multiple Preview builds](_images/multiple_base_preview_preview_builds.png)
+
+### Building a Preview from scratch after you've set a Base Preview
+
+Once you've set a Base Preview, new Previews build on that Base Preview by
+default. If you want a Preview to build from scratch, instead of a Base Preview:
+
+1. Go to the **Available to Build** section of the Repository Dashboard;
+2. Click into the drop-down next to the **Build Preview** button for the Preview
+   you'd like to build from scratch;
+3. Select the **Build with no base preview** option.
+
+Now your Preview will build from scratch!
+
+![Build with no base preview](_images/build_with_no_base_preview.png)
+
+## Administer Base Previews
```

Comment:
I think this whole section could use another pass. There are a lot of subsections that I think could be combined into one section with a screenshot and a few paragraphs to explain that you can select zero, one, or more base previews.